config_parser_test.cc

```diff
@@ -9,3 +12,184 @@ TEST(NginxConfigParserTest, SimpleConfig) {
 
   EXPECT_TRUE(success);
 }
+
+TEST(NginxConfigParserTest, VhostsConfig) {
+  NginxConfigParser parser;
+  NginxConfig out_config;
+
+  bool success = parser.Parse("vhosts_config", &out_config);
```

Comment:
Where is this `vhosts_config` file? I know your gitignore above ignores files without extensions, but this test doesn't appear to be repeatable without the config files used by each test?